bootstrap.py

```diff
@@ -369,13 +370,29 @@ def return_puppetenv_for_hg(hg_id):
     else:
         return 'production'
 
+def create_domain(domain,orgid,locid):
+    myurl = "https://" + options.foreman_fqdn + ":" + API_PORT + "/api/v2/domains"
+    domid = return_matching_foreman_key('domains', 'name="%s"' % domain, 'id', True)
+    if not domid:
+        jsondata = json.loads('{"domain": {"name": "%s", "organization_ids": [%s], "location_ids": [%s]}}' % (domain, orgid, locid))
+        print_running("Calling Foreman API to create domain %s associated with the org & location" % DOMAIN)
```

Comment:
that should be "domain", not "DOMAIN", while they do contain the same data, let's be consistent :)